lib/text_diff.ml

```diff
@@ -1,4 +1,8 @@
-type inline_conflict = { iorig : string option; inew : string option }
+type inline_conflict = {
+  iorig : string option * string option;
```

Comment:
I don't understand this new representation, what are each element of the pair?

Reply:
Sorry, this should've been a record. I think I will remove it altogether and restore the old representation.
I agree changing the representation to accommodate for "punctuation" (spaces, commas, etc.) should be done in a separate PR.